manifests/init.pp

```diff
@@ -10,11 +13,64 @@
 #   if false packages will be installed with "present"
 # @param  $version
```

Comment:
$version should be version. puppet strings fails when you have the $ 

Note that `puppet strings generate` spews many warnings that need to be fixed!